js/README.md

```diff
@@ -1,3 +1,88 @@
 # JS libraries
 
 TODO: split in a separate repo with npm `package.json`
+
+---
+## Descriptions
+JavaScript libraries based on the ETH implementation of the protocol.
+
+## Channel
+
+### Channel States:
```

Comment:
to Active, add "Or the channel is past the valid period but `channelWithdrawExpired` is not called yet to update the state to `Expired`"